uiTests/src/test/java/com/virtuslab/gitmachete/uitest/UITestSuite.java

```diff
@@ -73,6 +73,9 @@ public void ensureCorrectGraphTableRowCounts() {
 
   @Test
   public void fastForwardParentOfBranchAndEnsureCorrectRepositoryState() {
+    runJs("ide.soleOpenedProject().checkoutBranch('master')");
+    awaitIdle();
+    // `master` is the parent of `hotfix/add-trigger`. Let's fast-forward `master` to match `hotfix/add-trigger`.
     runJs("ide.soleOpenedProject().fastForwardParentToMatchBranch('hotfix/add-trigger')");
     awaitIdle();
 
```

Comment:
**TODO: the test is failing on `Assert.assertEquals(childBranchHash, parentBranchHash);`**